config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/customers/:id", to: "customers#show"
+  patch "/customers/:id", to: "customers#update"
```

Comment:
Not sure why you have a patch here, it's not wrong, but it wasn't asked for in the IC